client/pages/reviews.jsx

```diff
@@ -0,0 +1,288 @@
+import React from 'react';
+import Container from 'react-bootstrap/Container';
+import Row from 'react-bootstrap/Row';
+import Form from 'react-bootstrap/Form';
+import Col from 'react-bootstrap/Col';
+import activities from '../lib/activities';
+import Button from 'react-bootstrap/Button';
+
+const visitors = ['Everyone', 'History Buffs', 'Families', 'Casual Travelers', 'Teens & Adults', 'Outdoor Enthusiast', 'Nature Lovers'];
+
+export default class ReviewPage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      validated: false,
+      results: [],
+      isLoading: true,
+      rating: '',
+      activities: [],
+      visitors: [],
+      tips: '',
+      generalThoughts: '',
+      startDate: '',
+      endDate: ''
+    };
+    this.fileInputRef = React.createRef();
+    this.fetchData = this.fetchData.bind(this);
+    this.handleCheckBox = this.handleCheckBox.bind(this);
+    this.handleInputChange = this.handleInputChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  componentDidMount() {
+    this.fetchData();
+  }
+
+  handleInputChange(event) {
+    const value = event.target.value;
+    const name = event.target.name;
+    this.setState({
+      [name]: value
+    });
+  }
+
+  handleCheckBox(event) {
+    const target = event.target;
+    const name = event.target.name;
+    const array = this.state[name];
+    if (target.checked === true) {
+      this.setState({
+        name: array.push(event.target.id)
+      });
+    } else if (target.checked === false) {
+      const index = array.indexOf(event.target.id);
+      this.setState({
+        name: array.splice(index, 1)
+      });
+    }
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+    const form = event.currentTarget;
+    if (form.checkValidity() === false) {
+      event.stopPropagation();
+      this.setState({
+        validated: true
+      });
+      return;
+    }
+
+    const parkDetails = {
+      name: this.state.results.name,
+      imageUrl: this.state.results.wikiImage
+    };
+    const dates = [this.state.startDate, this.state.endDate];
+    let imageUrl = null;
+    if (this.fileInputRef.current.files[0]) {
+      imageUrl = this.fileInputRef.current.files[0];
+    }
+    const formData = new FormData();
+    formData.append('accountId', 1);
+    formData.append('image', imageUrl);
```

Comment:
This `imgUrl` variable name is misleading. We are actually dealing with a `File` object, so `image` or `imageFile` or `fileUpload` would be better names.